cmd/acraserver/acraserver.go

```diff
@@ -16,14 +16,15 @@ package main
 import (
 	"crypto/tls"
 	"flag"
+	"net/http"
+	_ "net/http/pprof"
```

Comment:
is it used?

Reply:
yes, here - https://github.com/cossacklabs/acra/pull/112/files/591904efc2140f6b44cbc2d3ab8441e4c72811eb#diff-963cfdc2dd5ec26b3315f2be5214f061R169
for debugging golang provide way to fetch profiling data over http